packages/bpk-component-badge/src/BpkBadge.js

```diff
@@ -42,7 +45,15 @@ const badgeTypeClassNames = {
   [BADGE_TYPES.outline]: getClassName('bpk-badge--outline'),
 };
 
-const BpkBadge = props => {
+type Props = {
+  type: ?string,
+  docked: ?boolean,
+  centered: ?boolean,
```

Comment:
Change the `?boolean` ones to `boolean`, because:

```
?boolean == true | false | null
boolean == true | false
```

So it's more accurate this way.